install_rpm_local.sh

```diff
@@ -1,115 +1,95 @@
-#!/bin/bash 
+#!/bin/bash
 
-# do the real building work
-# this script is executed on build VM
+# Do the real building work. This script is executed on build VM and
+# requires a working installation of CMake.
 
 set -x
 
 cd $work_dir
 
-sudo yum install -y rpmdevtools git
-sudo yum install -y wget
-sudo yum install -y tcl
-sudo yum install -y libuuid-devel
-sudo yum install -y xz-devel
-sudo zypper -n install rpmdevtools
-sudo zypper -n install git
-sudo zypper -n install wget
-sudo zypper -n install tcl
-sudo zypper -n install libuuid-devel
-sudo zypper -n install xz-devel
+command -v yum
 
+if [ $? -ne 0 ]
+then
+  sudo zypper -n install gcc gcc-c++ ncurses-devel bison glibc-devel libgcc_s1 perl \
+       make libtool libopenssl-devel libaio libaio-devel flex libcurl-devel \
+       pcre-devel systemtap-sdt-devel rpmdevtools git wget tcl libuuid-devel \
+       xz-devel sqlite3 sqlite3-devel pkg-config lua lua-devel rpm-build
 
-#wget http://pkgs.repoforge.org/flex/flex-2.5.35-0.8.el5.rfb.x86_64.rpm
-wget http://maxscale-jenkins.mariadb.com/x/flex-2.5.35-0.8.el5.rfb.x86_64.rpm
-#sudo yum install -
-sudo yum install flex-2.5.35-0.8.el5.rfb.x86_64.rpm -y --nogpgcheck 
-rm flex-2.5.35-0.8.el5.rfb.x86_64*
-. ~/check_arch.sh
+  cat /etc/*-release | grep "SUSE Linux Enterprise Server 11"
 
-yum --version
-if [ $? != 0 ] ; then
-	sudo zypper -n install rpm-build
-	zy=1
+  if [ $? -ne 0 ]
+  then
+      sudo zypper -n install libedit-devel
+  fi
 else
-	sudo yum install -y rpm-build createrepo yum-utils
-	zy=0
+  sudo yum clean all
+  sudo yum install -y --nogpgcheck gcc gcc-c++ ncurses-devel bison glibc-devel \
+       libgcc perl make libtool openssl-devel libaio libaio-devel libedit-devel \
+       libedit-devel libcurl-devel curl-devel systemtap-sdt-devel rpm-sign \
+       gnupg pcre-devel flex rpmdevtools git wget tcl openssl libuuid-devel xz-devel \
+       sqlite sqlite-devel pkgconfig lua lua-devel rpm-build createrepo yum-utils
```

Comment:
should --skip-broken Option be added? some packages are not obligatory for build and they can be unavailable in old distros. If any package in the list fails to be installed all 'yum' command fails. What is the option for zypper?